team-app/src/Search.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+
+export default Search = ({ setFilteredCards, teammates }) => {
+  const [searchInput, setSearchInput] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    try {
```

Comment:
i m just curious to see try, catch here. I need to research. is there a possibility for an exception here?

Reply:
added to check if invalid input breaks the code while filtering.